src/main/java/edu/ucla/library/bucketeer/verticles/FinalizeJobVerticle.java

```diff
@@ -47,7 +49,7 @@ public void start() throws Exception {
 
                     // If we have someone waiting on this result, let them know
                     if (slackHandle.isPresent()) {
-                        final String iiifURL = myConfig.getString(Config.IIIF_URL);
+                        final String iiifURL = getSimpleURL(myConfig.getString(Config.IIIF_URL));
```

Comment:
Sorry to chime in so late after you've been working on this, but this could be made simpler by using the static `URI.create` method to create a URI (not a URL) object (since all URLs are URIs), and then getting the hostname from it with `URI.getHost`. Since the IIIF URL is part of the application config, I think we can assume that parsing it will never throw a MalformedURLException -- and if it would, then we have bigger problems to deal with anyway.

Up to you if you'd like to change it since the change is so small.

Reply:
Switched to URI